### PR TITLE
Add timezone feature for ha-restarted-vms email notification

### PR DESCRIPTION
**Pull request by rkt6865**
Signed-off-by: rkt6865 <robert_terra@brown.edu>

## Add timezone feature so that time/date represented in email report could be "local time", if desired

- install tzdata to vmware/powerclicore image in template/powercli/Dockerfile
- add timezone field to vcconfig-ha-restarted-vms.json
- add function to handler/script.ps1 to verify/change time of event to "timezone" time
- tighten time constraints for searching events.  Previously, if more than one event (ie host crash) happened within the same day (12:00AM-12:00AM), duplicate entries would appear in the report. Now, as long as events happen after a 10 minute period from the previous event, duplicates are eliminated in report.
- update README.md to describe "TIMEZONE" file in config

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commmits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [x] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

List of Issues closed or resolved by this PR. Add multiple `Closes` keyword followed by the issue number (e.g. Closes #ISSUE-NUMBER)

* Closes #ISSUE-NUMBER

## Testing Verification

- tested against vSphere 6.7U3
- changed timezone to multiple timezones and verified correct output (including leaving timezone field blank)
- tested that duplicate entries are removed from report as long as "HA event(s)" do not happen within 10 minutes of the previous event.
- Please include any relevant screenshots and/or output as part of the testing

## Additional Information

* Any other details you wish to include or mention

If you have any questions/comments, feel free to reach out to team on Slack [#vcenter-event-broker-appliance](https://vmwarecode.slack.com/archives/CQLT9B5AA)

Thank you from the VEBA Team! 🥳

### CODE REVIEW

File: examples/powercli/ha-restarted-vms/handler/script.ps1 [hunk 1]
```diff
@@ -23,6 +23,11 @@ $eventnumber = 1000
 # get vCenter EventManager
 $si = get-view ServiceInstance
 $em = get-view $si.Content.EventManager
+# Create vmRestarted event begin/end times
+# End time = time of HA.ClusterFailoverActionCompletedEvent
+# Begin time = End time minus 10 minutes (arbitrary)
```
Comment (embano1) [inline examples/powercli/ha-restarted-vms/handler/script.ps1:28]: "arbitrary" as in it does not matter which value is picked or should this be configurable for the user?

Comment (rkt6865) [reply examples/powercli/ha-restarted-vms/handler/script.ps1:28]: Hi Michael,

"abitrary" as in I had to make an assumption that when an ESXi host fails, all VMs from that host will have been restarted within "10 minutes".  I could've chosen 5 mins, 15 mins, 1 hour, etc.  I am/was trying to eliminate duplicate entries on the report.  Previously the start/end times were 12AM "today" until 12AM "tomorrow".  If a host (or more than one host) crashed within the same day, the report would reflect the VMs from both events which was confusing.

**Comment by lamw** [comment]
@rkt6865 This is enhancing an existing function that was submitted and as part of that submission, its already included in the docs https://github.com/vmware-samples/vcenter-event-broker-appliance/commit/30dc878a8f1e59eb4acfefdc71df4f308eb63874#diff-1d930c291ce9e1bbd0d295a65b8b9709

**Comment by rkt6865** [comment]
Hi all,
Late yesterday I noticed that a single word should be changed.  When I pushed this latest code, for aesthetic purposes, I had changed one of the column names in the report from "ObjectName" to "VirtualMachine".  Unfortunately, I didn't change the sort, which still sorts on "ObjectName".
What's the best way to correct this?  Make the change, commit the change, (attempt) to squash the commits, re-push?  Or is there an easier/more direct way?
Thanks,
  - Bob

**Comment by embano1** [comment]
> What's the best way to correct this?  Make the change, commit the change, (attempt) to squash the commits, re-push?  Or is there an easier/more direct way?

Yes, change->commit->rebase (to squash) and (force) push to your PR branch is the right way.

**Comment by rkt6865** [comment]
Hi all,
I'm not sure if we are waiting on me for something or not.  I did neglect to comment after my push. My apologizes for that.
Thanks,
  - Bob

**Comment by rguske** [comment]
Hi @rkt6865 😁 , 
as far as I can see your last `push` looks good and the team will discuss all opened PRs and Issues very soon. PTO as well as VMworld preps postponed everything a little bit. Sorry for that and THANKS again for your contribution.

**Comment by rkt6865** [comment]
Hi Robert! Thanks for getting back.  No worries at all.  I figured there would be a lot of prep happening for VMworld.  Thanks again for reaching out!

**Comment by embano1** [comment]
Related #209

**Pull request closed without merging**